app/src/main/java/learning/android/tenmarks/com/androidlearning/SettingsActivity.java

```diff
@@ -1,10 +1,12 @@
 package learning.android.tenmarks.com.androidlearning;
 
 import android.app.Activity;
+import android.content.Intent;
 import android.media.audiofx.BassBoost;
```

Comment:
To order imports you can press CTRL + OPTION + O, this will remove unused imports and also set order to the imports. 
You can read more about this here: https://source.android.com/source/code-style.html#order-import-statements